tutorial/deploy_jaas_microk8s.rst

```diff
@@ -332,18 +336,101 @@ Verify that you can securely connect to JIMM with the following command:
 
 .. code:: bash
 
-    curl https://jimm.test.localhost/debug/info
+    curl https://test-jimm.localhost/debug/info
+
+Verify that you can login to your new controller with the Juju CLI.
+You should be presented with a message to login.
+
+.. code:: bash
+
+    juju login test-jimm.localhost:443 -c jimm-k8s
+    #Please visit https://iam.10.64.140.46.nip.io/iam-hydra/oauth2/device/verify and enter code <code> to log in.
 
 Using Your JIMM Deployment
 --------------------------
 
-Now that you have JIMM running you can browse our additional guides to start adding controllers and workloads.
+Now that you have JIMM running you can browse our additional guides to setup an admin user, add controllers and migrate existing workloads.
 
 - Setup your initial JIMM admin and configure permissions.
-- Learn how to add a new controller to JIMM.
+- :doc:`Learn how to add a new controller to JIMM.<../how-to/add_controller>`
 - Learn how to migrate models from existing controllers to JIMM.
 - Understand the difference between the ``juju``, ``jaas`` and ``jimmctl`` CLI tools.
 
+Common Issues
+-------------
+
+The following are some common issues that may arise especially after a reboot of your local machine.
+
+-------------------------------------
+JIMM server shows invalid certificate
+-------------------------------------
+Try ``curl https://jimm-test.localhost/debug/info``, if you receive an SSL certificate error then it's likely that the K8s ingress is no longer
+serving the correct TLS certificate. The following command can help verify this.
+
+.. code:: bash
+
+    openssl s_client -showcerts -servername test-jimm.localhost -connect test-jimm.localhost:443 </dev/null
+
+If the certificates CN (Common Name) is "Kubernetes Ingress Controller Fake Certificate" then the self-signed certificate is missing.
+Run the following to fix the issue.
+
+.. code:: bash
+
+    juju remove-relation ingress jimm-cert
+
+Wait for the relation to be removed by observing the output from ``juju status --relations --watch 2s``.
+
+.. code:: bash
+
+    juju relate ingress jimm-cert
+
+Try ``curl`` the server again the certificate issue should be resolved.
+
+----------------------------
+JIMM is not serving requests
+----------------------------
+
+If the JIMM server is not responding to requests run the following commands to check the logs.
+
+.. code:: bash
+
+    microk8s kubectl exec -it -n jimm jimm-0 -c jimm -- /charm/bin/pebble logs
+
+This will present the server logs and debug further.
+
+-------------------------------------------------
+JIMM can't communicate with the identity platform
+-------------------------------------------------
+
+If the JIMM server logs show an error similar to the following,
```

Comment:
suggestion: we need to decide what to call JIMM.. `JIMM controller`, `JIMM server` or just `JIMM` and use that throughout our documentation.

Reply:
I initially liked using the various option to help clarify to the reader that JIMM is just an API server rather than a Kubernetes controller. But to be consistent I think `JIMM controller` or `JIMM` (depending on where it's used) is probably best.